src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -934,7 +935,9 @@
     short-summary: Add an A record.
     examples:
         - name: Add an A record.
-          text: az network dns record-set a add-record -g MyResourceGroup -z www.mysite.com -n MyRecordSet -a MyIpv4Address
+          text: |
+            az network dns record-set a add-record -g MyResourceGroup -z www.mysite.com\\
```

Comment:
Either put the example back on a single line (which is OK, shouldn't have called these out) or add whitespace before `\\` and indent all continuation lines.

Reply:
applied the latter in formatting